### PR TITLE
feat: Add create-react-native-library

### PR DESCRIPTION
**Pull request by LazyAfternoons**
- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

File: radar/tools/create-react-native-library.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Creation of React Native libraries for iOS and Android which use device-specific features like interacting with the keystore on Android or the keychain on iOS.
+- Creation of React Native libraries for iOS and Android which implement performance-critical tasks more efficiently through native code integration, like cryptography operations.
+- Creation of React Native libraries for iOS and Android which leverage native UI counterparts on specific platforms, like a native map view or a native video player.
+
+## Reference of usage in our organization
+
+- [IO React Native Crypto](https://github.com/pagopa/io-react-native-crypto)
+- [IO React Native Login Utils](https://github.com/pagopa/io-react-native-login-utils)
+- [IO React Native JWT](https://github.com/pagopa/io-react-native-jwt)
```
Comment (CrisTofani) [inline radar/tools/create-react-native-library.md:29]:
```suggestion
- [IO React Native JWT](https://github.com/pagopa/io-react-native-jwt)
- [IO App Design System](https://github.com/pagopa/io-app-design-system)
```

**Review by CrisTofani — CHANGES_REQUESTED** [review]

File: radar/tools/create-react-native-library.md (hunk 1 above)
Comment (LazyAfternoons) [reply radar/tools/create-react-native-library.md:29]: Done in [aa47a68](https://github.com/pagopa/technology-radar/pull/52/commits/aa47a68bef90f71f00516281884253d411e7d9e5), thank you

File: radar/tools/create-react-native-library.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+quadrant: "tools"
+tags: [mobile, react-native, swift, kotlin, java, objective-c, c++]
+---
+
+[create-react-native-library](https://callstack.github.io/react-native-builder-bob/) is a CLI for scaffolding React Native libraries for iOS and Android. It simplifies the process of creating, building, testing, and publishing React Native libraries by providing a CLI that leverages `react-native-builder-bob`.
+
+#### Key features:
+
+1. **Simplicity**: Provides a CLI which guides you in the creation of different types of React Native libraries.
+2. **Toolchain**: Supports Typescript, ESLint, Prettier, TypeScript, Lefthook and Release It out of the box.
```
Comment (hevelius) [inline radar/tools/create-react-native-library.md:13]: Is there a duplication for Typescript?

**Review by hevelius — CHANGES_REQUESTED** [review]

File: radar/tools/create-react-native-library.md (hunk 2 above)
Comment (LazyAfternoons) [reply radar/tools/create-react-native-library.md]: Nice catch, fixed in 8921df9b, thanks.

**Review by hevelius — APPROVED** [review]
LGTM

**Pull request merged**